amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4307,26 +4321,100 @@ definitions:
     properties:
       name:
         type: string
+        description: |
+          The valid names are:
+          * retain
+          * cutoff
+          * accession
+          * transfer
+          * destroy
       periodAmount:
         type: integer
+        description: |
+          This property only applicable for the below period values.
+          * day
+          * month
+          * quarter
+          * week
+          * duration
+          * year
```

Comment:
I think this is missing `xmlDuration` and an explanation of how to use it (or a canonical link).

Reply:
Here the duration property refers to xmlDuration only. we have mentioned the same in the documentation as well. We have added more detailed explanation for how to add value for xml duration.